package.json

```diff
@@ -51,6 +51,7 @@
   },
   "dependencies": {
     "ember-tether": "^1.0.0-beta.0",
+    "metismenu": "^2.7.0",
```

Comment:
You have added metismenu here but it is already on vendor/metisMenu. I think this can gone.